src/Core/src/Platform/iOS/SearchBarExtensions.cs

```diff
@@ -10,9 +10,25 @@ public static class SearchBarExtensions
 		internal static UITextField? GetSearchTextField(this UISearchBar searchBar)
 		{
 			if (OperatingSystem.IsIOSVersionAtLeast(13))
+			{
 				return searchBar.SearchTextField;
-			else
-				return searchBar.GetSearchTextField();
+			}
```

Comment:
If iOS < 13 before, this just threw `StackOverflowException`? Let me know if I'm missing something, I ported a Swift example to C# instead.